rotkehlchen/globaldb/utils.py

```diff
@@ -30,3 +34,20 @@ def globaldb_get_setting_value(cursor: 'DBCursor', name: str, default_value: int
         return default_value
 
     return int(result[0][0])
+
+
+def toggle_token_spam_protocol(
```

Comment:
The name is wrong. Toggle means to toggle ... aka to flip. If it was false, set to true, if it was true set to false. You explicitly set it, so this is a setter

```suggestion
def set_token_spam_protocol(
```